buildSrc/src/gen-execution.kt

```diff
@@ -150,6 +150,60 @@ fun LangData.execution(nickname: String, configName: String, outDir: File) {
 	}
 	""".trimIndent()
 
+	@Language("java")
+	val runConfigEditorJava = """
+	package $basePackage.execution;
+	
+	import com.intellij.openapi.fileChooser.FileChooserDescriptorFactory;
+	import com.intellij.openapi.project.Project;
+	import ${basePackage}.TTBundle;
+	import ${basePackage}.execution.${languageName}RunConfiguration;
+	import ${basePackage}.execution.ui.InterpretedRunConfigurationEditorImpl;
+	import $basePackage.${languageName}FileType;
+	import org.jetbrains.annotations.NotNull;
+
+	import javax.naming.ConfigurationException;
+	import java.lang.reflect.Executable;
+
+	public class ${languageName}RunConfigurationEditor extends InterpretedRunConfigurationEditorImpl<${languageName}RunConfiguration> {
+
+		private ${languageName}RunConfiguration configuration;
+		private Project project;
+
+		public ${languageName}RunConfigurationEditor(${languageName}RunConfiguration configuration, Project project) {
+			super(project);
+			this.configuration = configuration;
+			this.project = project;
+			targetFileField.addBrowseFolderListener(TTBundle.message("$nickname.ui.run-config.select-$nickname-file"),
+				TTBundle.message("$nickname.ui.run-config.select-$nickname-file.description"),
+				project,
+				FileChooserDescriptorFactory.createSingleFileDescriptor(${languageName}FileType.INSTANCE));
+			exePathField.addBrowseFolderListener(TTBundle.message("$nickname.ui.project.select-compiler"),
+				TTBundle.message("$nickname.ui.project.select-compiler.description"),
+				project,
+				FileChooserDescriptorFactory.createSingleFileOrExecutableAppDescriptor());
+			resetEditorFrom(configuration);
+		}
+
+		@Override
+		protected void resetEditorFrom(@NotNull ${languageName}RunConfiguration s) {
+			super.resetEditorFrom(s);
+			exePathField.setText(s.${configName}Executable);
+		}
+
+		@Override
+		protected void applyEditorTo(@NotNull ${languageName}RunConfiguration s) {
+			try {
+				super.applyEditorTo(s);
+			} catch (ConfigurationException e) {
+				e.printStackTrace();
```

Comment:
How about adding the Exception to the `throws` List?

Reply:
![image](https://user-images.githubusercontent.com/10379980/66860255-d9b31c80-ef5a-11e9-9430-06b2461b7669.png)
getting this

Reply:
Ok. I'll try come up with a better solution for this